extensions/rich_text_components/Video/directives/oppia-noninteractive-video.component.spec.ts

```diff
@@ -172,7 +172,7 @@ describe('NoninteractiveVideo', () => {
 
   it('should not update video view if atleast one video parameters' +
     ' is undefined', () => {
```

Comment:
Instead of `is undefined` maybe `is empty` would be better now.

Reply:
done